adb_graphics/default_specs.yml

```diff
@@ -163,13 +188,19 @@ cloudcover:
     ncl_name: TCDC_P0_L10_GLC0
     title: Total Cloud Cover
 cref: # Composite reflectivity
-  sfc: &refl
-    clevs: !!python/object/apply:numpy.arange [5, 76, 5]
-    cmap: NWSReflectivity
-    colors: cref_colors
+  sfc:
+    <<: *refl
     ncl_name: REFC_P0_L10_GLC0
-    ticks: 5
-    unit: dB
+    title: Composite Reflectivity
+cpofp:
```

Comment:
What do these letters mean?

Reply:
Categorical Percentage Of Frozen Precipitation.  Seems to be standard at least for NCEP and GSL.

I moved the name comment up a line from "sfc:" line below it.